docs/UserGuide.md

```diff
@@ -6,7 +6,8 @@
 
 # WedLinker User Guide
 
-WedLinker is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+WedLinker is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). </br>
+If you can type fast, WedLinker can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
I feel like this can still be more user-centric and targeted towards wedding planners, right now it still seems a bit generic? Maybe can talk about how its designed specifically for wedding planners to efficiently plan weddings